eclipse-collections-api/src/main/java/org/eclipse/collections/api/factory/map/sorted/MutableSortedMapFactory.java

```diff
@@ -22,6 +22,8 @@
      */
     <K, V> MutableSortedMap<K, V> empty();
 
+    <K, V> MutableSortedMap<K, V> empty(Comparator<? super K> comparator);
```

Comment:
This will require a `since` tag of 12.0 since it is an API change without a default implementation.